src/parser.cpp

```diff
@@ -435,7 +441,7 @@ namespace ada::parser {
           // - url is special and c is U+005C (\)
           else if (pointer == pointer_end || *pointer == '/' || *pointer == '?' || (url.is_special() && *pointer == '\\')) {
             // then decrease pointer by 1, and then:
-            pointer--;
+            // pointer--;
```

Comment:
If this is true, can we remove the comment?

Reply:
Removed.